code/datums/recipe.dm

```diff
@@ -38,6 +38,7 @@
 	var/list/items // example: =list(/obj/item/crowbar, /obj/item/welder) // place /foo/bar before /foo
 	var/result //example: = /obj/item/reagent_containers/food/snacks/donut
 	var/time = 100 // 1/10 part of second
+	var/duplicate = TRUE // Whether or not an upgraded kitchen machine will create more products using the same amount of ingredients
```

Comment:
Could you autodoc this comment? I know it wouldn't be consistent with the rest of the comments but it's still better practice.